compact/fields/beamline_10x100.xml

```diff
@@ -44,7 +44,7 @@
     <constant name="Q1EF_GradientMax"  value="2.127596364*tesla/m"/>
 
     <constant name="B0PF_Bmax"  value="1.1840539*tesla"/>
-    <constant name="B0APF_Bmax" value="1.1588921*tesla"/> <!-- need to check with Scott on this one -->
+    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/> <!-- need to check with Scott on this one -->
```

Comment:
Did you check with Scott yet? ;-) If so:
```suggestion
    <constant name="B0APF_Bmax" value="-1.1588921*tesla"/>
```

Reply:
Done.